GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -149,6 +149,10 @@ public abstract class GVFSPlatformConstants
             /// the upgrade verb is running.
             /// </summary>
             public abstract bool SupportsUpgradeWhileRunning { get; }
+            public abstract string UpgradeInstallAdvice { get; }
+            public abstract string ElevatedCommandPrompt { get; }
+            public abstract string UpgradeConfirmCommand { get; }
+            public abstract string StartServiceCommand { get; }

```

Comment:
Naming suggestion:  Add "Message" to the end of each of these new property names.  

Previously they were in `UpgradeVerbMessages` (which made it clear these are user facing messages) but now glancing at the code someone might think `StartServiceCommand` is something VFS4G could call.